### PR TITLE
Stop generating group_map_params.ml

### PR DESCRIPTION
**Pull request by martyall**
current `gen.exe` benchmark: `0.12636184692382812ms`

### CODE REVIEW

**Comment by martyall** [comment]
!ci-build-me

File: src/lib/crypto_params/crypto_params.ml [hunk 1]
```diff
@@ -6,7 +6,12 @@ module Tock = struct
   module Full = Pickles.Impls.Wrap
   module Run = Pickles.Impls.Wrap
 
-  let group_map_params () = Lazy.force Group_map_params.params
+  let group_map_params () =
+    Lazy.force
```
Comment (mrmr1993) [inline src/lib/crypto_params/crypto_params.ml:10]: Doing `Lazy.force` immediately on a `lazy` is redundant. Remove both.

File: src/lib/crypto_params/crypto_params.ml [hunk 2]
```diff
@@ -6,7 +6,10 @@ module Tock = struct
   module Full = Pickles.Impls.Wrap
   module Run = Pickles.Impls.Wrap
 
-  let group_map_params () = Lazy.force Group_map_params.params
+  let group_map_params =
```
Comment (mrmr1993) [inline src/lib/crypto_params/crypto_params.ml:9]: This should probably still be a function to avoid doing work immediately at startup

Comment (martyall) [reply src/lib/crypto_params/crypto_params.ml:9]: I just benchmarked all of these generator files using the same technique:

```ocaml
let main = 
  let start = Time.now () in
  (* body of main function *)
    let e = Time.now () in
  let () = print_endline (Float.to_string @@ Time.Span.to_ms (Time.diff e start)) in
  exit 0
  ```
  
  each PR's description contains the benchmark, and this one takes less than `.2ms` to compute. I would argue that this kind of premature optimization isn't necessary. Also assuming the `lazy` semantics of rust, if you want the function to stay I feel like we should keep the value listed as `lazy` no?

**Comment by martyall** [comment]
!ci-build-me

**Comment by martyall** [comment]
!ci-build-me

**Comment by martyall** [comment]
Closing in favor of #15902

**Pull request closed without merging**